pyanaconda/modules/subscription/satellite.py

```diff
@@ -0,0 +1,161 @@
+#
+# Satellite support purpose library.
+#
+# Copyright (C) 2020 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+
+import os
+
+from requests import RequestException
+
+from pyanaconda.core import constants, util
+
+# FIXME: move this to core ? use different user agent
+#        without product version information ?
+from pyanaconda.payload.dnf.utils import USER_AGENT
```

Comment:
The current implementation does:

`USER_AGENT = "%s (anaconda)/%s" % (productName, productVersion)`

Eq. as far as I can tell nothing payload specific - what about moving the definition somewhere to core so that both payload and subscription code can use that ? 

I would expect all Anaconda interaction with web services to present the same user agent, so that should not be a problem.